language/tools/move-cli/tests/cli_testsuite.rs

```diff
@@ -8,8 +8,8 @@ use std::path::Path;
 pub const CLI_BINARY: &str = "../../../target/debug/move-cli";
 pub const CLI_TESTSUITE_DIR: &str = "tests/testsuite";
 
-fn run_all(args_path: &Path) -> datatest_stable::Result<()> {
-    Ok(test::run_one(args_path, CLI_BINARY)?)
+fn test_run_all(args_path: &Path) -> datatest_stable::Result<()> {
+    Ok(test::run_all(args_path, CLI_BINARY)?)
 }
 
-datatest_stable::harness!(run_all, CLI_TESTSUITE_DIR, r"args.txt$");
+datatest_stable::harness!(test_run_all, CLI_TESTSUITE_DIR, r"args.txt$");
```

Comment:
Now that we have `run_all`, I wonder if it would work to kill lines 11--15 and just do something like

```
#[cfg(test)]
fn go() {
  test::run_all(CLI_TESTSUITE_DIR, CLI_BINARY)?
}
```

Reply:
Thanks for the suggestion! I'll fix it.